cmd/metrics-server/app/options/options_test.go

```diff
@@ -149,3 +149,46 @@ func TestKubeletConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestValidate(t *testing.T) {
+	for _, tc := range []struct {
+		name               string
+		options            *Options
+		expectedErrorCount int
+	}{
+		{
+			name:               "Default options should pass validate",
+			options:            NewOptions(),
+			expectedErrorCount: 0,
+		},
+		{
+			name: "Cannot use both --kubelet-ca-file and --deprecated-kubelet-completely-insecure",
+			options: &Options{
+				DeprecatedCompletelyInsecureKubelet: true,
+				KubeletCAFile:                       "a",
+			},
+			expectedErrorCount: 1,
+		},
+		{
+			name: "Cannot use both --kubelet-ca-file and --kubelet-insecure-tls",
+			options: &Options{
+				InsecureKubeletTLS: true,
+				KubeletCAFile:      "a",
+			},
+			expectedErrorCount: 1,
+		},
+		{
+			name: "use both --kubelet-ca-file and --kubelet-key-file",
```

Comment:
Please add test that sets:
* only `--kubelet-ca-file`  flag and is expected to fail
* only `--kubelet-key-file` flag and is expected to fail